axelor-account/src/main/java/com/axelor/apps/account/service/AccountService.java

```diff
@@ -68,4 +84,56 @@ public BigDecimal computeBalance(Account account, int balanceType) {
       return BigDecimal.ZERO;
     }
   }
+
+  public List<MoveLine> searchMoveLinesByAccountAndAnalytic(Account account, boolean isWithAnalytic)
+      throws Exception {
+    String analyticFilter = "";
+    if (isWithAnalytic) {
+      analyticFilter =
+          "(moveLine.analyticDistributionTemplate is not null OR (SELECT COUNT(0) FROM AnalyticMoveLine  analyticMoveLine WHERE analyticMoveLine.moveLine = moveLine.id) > 0)";
```

Comment:
The line is too long, make lines that are less than 100 characters (as specified in google java format)